src/components/Menu/Menu.module.scss

```
@@ -0,0 +1,138 @@
+.menu-v3 {
```

This is an outdated class from the Admin UI. Remove the `-v3` from all css classes.
```suggestion
.menu {
```